add delete ortho

FredericBlum opened the pull request:
@jaimerafaelms We are on a new branch 'orthography'. I have reverted your revert, so to say, to restore the previous files you had for the first 26 languages. As I had presented already in the other PR, there is still some problems lying around. Probably easy fixes, given that you have the sources and language-specific knowledge.

To check that all the changes have been implemented correctly, please run the `cldfbench` command yourself, as we had shown in the tutorials. The `Transripton.md` file then automatically updates.

jaimerafaelms committed:
add delete ortho

FredericBlum committed:
re-run conversion

jaimerafaelms commented:
¡Gracias Frederic! Ahora puedo ver todas las ortografías actualizadas en el nuevo branch. Ahora voy a revisar los errores que quedan.

jaimerafaelms committed:
correcciones

- corrección de errores en las correspondencias entre lenguas
- correcciones de ideófonos en raw.tsv

jaimerafaelms committed:
correcciones2

faltó corregir Wayana

jaimerafaelms commented:
Frederic, ya corregí los errores de correspondencias grafemas-fonemas. I have added the changes in commits. Me parece que ya todo está corregido. No encuentro en tus tutoriales cómo usar el comando cldfbench; por eso no puedo actualizar TRANSCRIPTION.md.

Preguntas puntuales sobre mis correcciones:
1. Cuando un símbolo no tiene correspondencia fonética (por ejemplo, "-"), he colocado "NULL" en la columna del fonema, ¿esto es correcto?
2. En dos lenguas TRANSCRIPTION.md reporta errores en símbolos que sí son correctos (karo.tsv y saramaccan.tsv). Justo en estas lenguas hay fonemas que no son símbolos convencionales del IPA (mᵇ, nᵈ, nᶡ, ŋᵍ, tʙ), ¿tal vez reporta error porque no son símbolos convencionales?

Te agradecería mucho si me pudieses enseñar el uso del comando cldfbench, o en todo caso si pudieses corroborar que TRANSCRIPTION.md ya no reporta errores. Estoy probando en VSCodium y no sé por qué no me funciona el código que me mandaste (cldfbench lexibank.makecldf lexibank_kindbergashaninca.py --clts-version=v2.3.0 --glottolog-version=v5.1 --concepticon=../../cldf-resources/concepticon-data/).

FredericBlum commented:
>     1. Cuando un símbolo no tiene correspondencia fonética (por ejemplo, "-"), he colocado "NULL" en la columna del fonema, ¿esto es correcto?

En caso de "-" utilizamos /+/ para marcar la separación de morphemas. Para los demás, NULL está perfecto.


>     2. En dos lenguas TRANSCRIPTION.md reporta errores en símbolos que sí son correctos (karo.tsv y saramaccan.tsv). Justo en estas lenguas hay fonemas que no son símbolos convencionales del IPA (mᵇ, nᵈ, nᶡ, ŋᵍ, tʙ), ¿tal vez reporta error porque no son símbolos convencionales?

Necesito la descripción de esos fonemas para proponer un simbolo adecuado de CLTS. Estoy seguro que lo vamos a encontrar. Pero por solo ver los simbolos sin nada más es dificil.

> Te agradecería mucho si me pudieses enseñar el uso del comando cldfbench, o en todo caso si pudieses corroborar que TRANSCRIPTION.md ya no reporta errores. Estoy probando en VSCodium y no sé por qué no me funciona el código que me mandaste (cldfbench lexibank.makecldf lexibank_kindbergashaninca.py --clts-version=v2.3.0 --glottolog-version=v5.1 --concepticon=../../cldf-resources/concepticon-data/).

Perdón, el código correcto va asi:
`cldfbench lexibank.makecldf lexibank_ideobank.py --clts-version=v2.3.0 --glottolog-version=v5.1 --concepticon-version=v3.4.0)`

Eso está asumiendo que has instalado cldfbench, y luego corrido `cldfbench catconfig` (preparando los catálogos de referencia).

jaimerafaelms commented:
> Eso está asumiendo que has instalado cldfbench, y luego corrido cldfbench catconfig (preparando los catálogos de referencia).

He realizado estos pasos, pero aún me sale error en el código que me has mandado. No veo esta información en los tutoriales para poder orientarme, googleando al respecto tampoco encuentro respuesta. ¿Crees que podamos tener una reunión rápida  para que pueda correr el código correctamente (puede ser 9 o 10 hora peruana, o a otra hora que te sea conveniente)?

FredericBlum commented:
Por favor, copia el codigo que corres y también el error que te sale.

jaimerafaelms commented:
El código 

(lexibank) PS C:\Users\Jaime\Documents\GitHub\ideobank> cldfbench lexibank.makecldf lexibank_ideobank.py --clts-version=v2.3.0 --glottolog-version=v5.1 --concepticon-version=v3.4.0 

El error:

usage: cldfbench [-h] [--log-level LOG_LEVEL] [-z FIELD_SIZE_LIMIT] [--random-seed RANDOM_SEED] COMMAND ...
cldfbench: error: argument COMMAND: invalid choice: 'lexibank.makecldf' (choose from catconfig, catinfo, catupdate, check, ci, cldfreadme, diff, download, geojson, info, ls, makecldf, media, new, readme, run, stub, zenodo, zenodo.download)
(lexibank) PS C:\Users\Jaime\Documents\GitHub\ideobank>

FredericBlum commented:
Probabltemente falta `pip install pylexibank`

jaimerafaelms commented:
Cuando hago correr `pip install pylexibank` todas las líneas siguientes empiezan con "Requirement already satisfied:"

Después de esto, he vuelto a colocar a colocar el código `cldfbench lexibank.makecldf lexibank_ideobank.py --clts-version=v2.3.0 --glottolog-version=v5.1 --concepticon-version=v3.4.0` y sale lo siguiente:

> Traceback (most recent call last):
>   File "<frozen runpy>", line 198, in _run_module_as_main
>   File "<frozen runpy>", line 88, in _run_code
>   File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Scripts\cldfbench.exe\__main__.py", line 6, in <module>
>     sys.exit(main())
>              ~~~~^^
>   File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\cldfbench\__main__.py", line 73, in main
>     stack.enter_context(
>     ~~~~~~~~~~~~~~~~~~~^
>         cls(path, getattr(args, name + '_version', None))),
>         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "C:\Users\Jaime\AppData\Local\Python\pythoncore-3.14-64\Lib\contextlib.py", line 530, in enter_context
>     result = _enter(cm)
>   File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\cldfcatalog\catalog.py", line 73, in __enter__
>     self.checkout(self.tag)
>     ~~~~~~~~~~~~~^^^^^^^^^^
>   File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\cldfcatalog\repository.py", line 129, in checkout
>     return self.repo.git.checkout(spec)
>            ~~~~~~~~~~~~~~~~~~~~~~^^^^^^
>   File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\git\cmd.py", line 1003, in <lambda>
>     return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
>                                    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
>   File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\git\cmd.py", line 1616, in _call_process
>     return self.execute(call, **exec_kwargs)
>            ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
>   File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\git\cmd.py", line 1406, in execute
>     raise GitCommandError(redacted_command, status, stderr_value, stdout_value)
> git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
>   cmdline: git checkout v5.1
>   stderr: 'error: The following untracked working tree files would be overwritten by checkout:
>         .gitattributes
>         .gitignore
>         .zenodo.json
>         CHANGES.md
>         CONTRIBUTING.md
>         CONTRIBUTORS.md
>         LICENSE
>         MANIFEST.in
>         README.md
>         RELEASING.md
>         SOURCES.md
>         config/README.md
>         config/aes_sources.ini
>         config/aes_status.ini
>         config/document_types.ini
>         config/editors.ini
>         config/language_types.ini
>         config/languoid_levels.ini
>         config/macroareas.ini
>         config/macroareas/voronoi/africa.geojson
>         config/macroareas/voronoi/australia.geojson
>         config/macroareas/voronoi/eurasia.geojson
>         config/macroareas/voronoi/north_america.geojson
>         config/macroareas/voronoi/papunesia.geojson
>         config/macroareas/voronoi/south_america.geojson
>         config/med_types.ini
>         config/publication.ini
>         faq.md
>         languoids/README.md
>         languoids/dialects.md
>         languoids/dialects_3.md
>         languoids/dialects_a.md
>         languoids/dialects_b.md
>         languoids/dialects_c.md
>         languoids/dialects_d.md
>         languoids/dialects_e.md
>         languoids/dialects_f.md
>         languoids/dialects_g.md
>         languoids/dialects_h.md
>         languoids/dialects_i.md
>         languoids/dialects_j.md
>         languoids/dialects_k.md
>         languoids/dialects_l.md
>         languoids/dialects_m.md
>         languoids/dialects_n.md
>         languoids/dialects_o.md
>         languoids/dialects_p.md
>         languoids/dialects_q.md
>         languoids/dialects_r.md
>         languoids/dialects_s.md
>         languoids/dialects_t.md
>         languoids/dialects_u.md
>         languoids/dialects_v.md
>         languoids/dialects_w.md
>         languoids/dialects_x.md
>         languoids/dialects_y.md
>         languoids/dialects_z.md
>         languoids/families.md
>         languoids/families_a.md
>         languoids/families_b.md
>         languoids/families_c.md
>         languoids/families_d.md
>         languoids/families_e.md
>         languoids/families_f.md
>         languoids/families_g.md
>         languoids/families_h.md
>         languoids/families_i.md
>         languoids/families_j.md
>         languoids/families_k.md
>         languoids/families_l.md
>         languoids/families_m.md
>         languoids/families_n.md
>         languoids/families_o.md
>         languoids/families_p.md
>         languoids/families_q.md
>         languoids/families_r.md
>         languoids/families_s.md
>         languoids/families_t.md
>         languoids/families_u.md
>         languoids/families_v.md
>         languoids/families_w.md
>         languoids/families_x.md
>         languoids/families_y.md
>         languoids/families_z.md
>         languoids/glottocodes.json
>         languoids/languages.md
>         languoids/languages_a.md
>         languoids/languages_b.md
>         languoids/languages_c.md
>         languoids/languages_d.md
>         languoids/languages_e.md
>         languoids/languages_f.md
>         languoids/languages_g.md
>         languoids/languages_h.md
>         languoids/languages_i.md
>         languoids/languages_j.md
>         languoids/languages_k.md
>         languoids/languages_l.md
>         languoids/languages_m.md
>         languoids/languages_n.md
>         languoids/languages_o.md
>         languoids/languages_p.md
>         languoids/languages_q.md
>         languoids/languages_r.md
>         languoids/languages_s.md
>         languoids/languages_t.md
>         languoids/languages_u.md
>         languoids/languages_v.md
>         languoids/languages_w.md
>         languoids/languages_x.md
>         languoids/languages_y.md
>         languoids/languages_z.md
>         languoids/superseded.csv
>         languoids/tree/abin1243/md.ini
>         languoids/tree/abis1238/md.ini
>         languoids/tree/abkh1242/abkh1243/abaz1241/ashk1247/md.ini
>         languoids/tree/abkh1242/abkh1243/abaz1241/bezs1238/md.ini
>         languoids/tree/abkh1242/abkh1243/abaz1241/md.ini
>         languoids/tree/abkh1242/abkh1243/abaz1241/tapa1256/md.ini
>         languoids/tree/abkh1242/abkh1243/abkh1244/abzh1238/md.ini
>         languoids/tree/abkh1242/abkh1243/abkh1244/bzyb1238/md.ini
>         languoids/tree/abkh1242/abkh1243/abkh1244/md.ini
>         languoids/tree/abkh1242/abkh1243/abkh1244/samu1242/md.ini
>         languoids/tree/abkh1242/abkh1243/md.ini
>         languoids/tree/abkh1242/circ1239/adyg1241/abad1240/md.ini
>         languoids/tree/abkh1242/circ1239/adyg1241/bezh1247/md.ini
>         languoids/tree/abkh1242/circ1239/adyg1241/md.ini
>         languoids/tree/abkh1242/circ1239/adyg1241/natu1243/md.ini
>         languoids/tree/abkh1242/circ1239/adyg1241/shap1240/md.ini
>         languoids/tree/abkh1242/circ1239/adyg1241/temi1249/md.ini
>         languoids/tree/abkh1242/circ1239/adyg1241/xaku1238/md.ini
>         languoids/tree/abkh1242/circ1239/kaba1278/grea1271/md.ini
>         languoids/tree/abkh1242/circ1239/kaba1278/md.ini
>         languoids/tree/abkh1242/circ1239/md.ini
>         languoids/tree/abkh1242/md.ini
>         languoids/tree/abkh1242/ubyk1235/md.ini
>         languoids/tree
> error: The following untracked working tree files would be removed by checkout:
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/arsi1238/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/bora1272/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/bora1272/yaab1234/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/gabr1255/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/guji1239/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/kere1294/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/saku1238/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/sala1277/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/orma1241/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/orma1241/muny1242/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/orma1241/nucl1702/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/orma1241/waat1239/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/waat1238/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/bare1280/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/nort3272/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/nort3272/raya1242/md.ini
>         languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/nort3272/well1239/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/hass1238/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1239/algi1247/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1239/cons1245/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1239/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1239/oran1236/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1240/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/anda1287/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/jude1265/kjal1234/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/jude1265/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/jude1265/wqal1234/md.ini
>         languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/malt1259/malt1254/gozo1238/md.ini
>         la
> Aborting'

FredericBlum commented:
Que te sale con los comandos

``` shell
cldfbench catupdate
cldfbench catinfo
```

? Si catupdate actualiza algunas versiones, intenta correr el cldfbench de nuevo.

jaimerafaelms commented:
Al poner `cldfbench catupdate` sale lo siguiente:

```
INFO    glottolog: fetch origin/master 
INFO    concepticon: fetch origin/master 
INFO    clts: fetch origin/master    
```

Después al poner `cldfbench catinfo` sale esto:

```
glottolog - https://github.com/glottolog/glottolog

local clone: C:\Users\Jaime\AppData\Local\cldf\cldf\glottolog
config at: C:\Users\Jaime\AppData\Local\cldf\cldf\catalog.ini
versions:
  v5.2.1            release <version>
  v5.2              release 5.2
  v5.1              release 5.1
  v5.0              release 5.0
  v4.8              release 4.8
API: pyglottolog 3.16.0


concepticon - https://github.com/concepticon/concepticon-data

local clone: C:\Users\Jaime\AppData\Local\cldf\cldf\concepticon
config at: C:\Users\Jaime\AppData\Local\cldf\cldf\catalog.ini
versions:
  v3.4.0      release 3.4.0
  v3.3.0      v3.3.0 (#1445)
  v3.2.0      release 3.2.0
  v3.1.0      release 3.1.0
  v3.0.0      release 3.0
API: pyconcepticon 3.1.0


clts - https://github.com/cldf-clts/clts

local clone: C:\Users\Jaime\AppData\Local\cldf\cldf\clts
config at: C:\Users\Jaime\AppData\Local\cldf\cldf\catalog.ini
versions:
  v2.3.0 2.3.0 release
  v2.2.0 release candidate for v2.2.0 (#128)
  v2.1.0 added CI badge
  v2.0.0 2.0.0 release
  v1.4.1 CLTS recreated with pyclts 2.0
API: pyclts 3.2.0
```

Luego hago correr `cldfbench catconfig` y sale esto:

```
INFO    Clone of glottolog/glottolog exists at C:\Users\Jaime\AppData\Local\cldf\cldf\glottolog - skipping
INFO    Clone of concepticon/concepticon-data exists at C:\Users\Jaime\AppData\Local\cldf\cldf\concepticon - skipping
INFO    Clone of cldf-clts/clts exists at C:\Users\Jaime\AppData\Local\cldf\cldf\clts - skipping
INFO    Config written to C:\Users\Jaime\AppData\Local\cldf\cldf\catalog.ini
```
Finalmente, hago correr el código `cldfbench lexibank.makecldf lexibank_ideobank.py --clts-version=v2.3.0 --glottolog-version=v5.1 --concepticon-version=v3.4.0` pero sale el mismo mensaje largo de mi post anterior. Lo repito aquí dentro de ``` este formato ``` para que las líneas sean más legibles:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Scripts\cldfbench.exe\__main__.py", line 6, in <module>
    sys.exit(main())
             ~~~~^^
  File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\cldfbench\__main__.py", line 73, in main
    stack.enter_context(
    ~~~~~~~~~~~~~~~~~~~^
        cls(path, getattr(args, name + '_version', None))),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jaime\AppData\Local\Python\pythoncore-3.14-64\Lib\contextlib.py", line 530, in enter_context
    result = _enter(cm)
  File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\cldfcatalog\catalog.py", line 73, in __enter__
    self.checkout(self.tag)
    ~~~~~~~~~~~~~^^^^^^^^^^
  File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\cldfcatalog\repository.py", line 129, in checkout
    return self.repo.git.checkout(spec)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\git\cmd.py", line 1003, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
                                   ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\git\cmd.py", line 1616, in _call_process
    return self.execute(call, **exec_kwargs)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jaime\Documents\GitHub\ideobank\venv\lexibank\Lib\site-packages\git\cmd.py", line 1406, in execute
    raise GitCommandError(redacted_command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git checkout v5.1
  stderr: 'error: The following untracked working tree files would be overwritten by checkout:
        .gitattributes
        .gitignore
        .zenodo.json
        CHANGES.md
        CONTRIBUTING.md
        CONTRIBUTORS.md
        LICENSE
        MANIFEST.in
        README.md
        RELEASING.md
        SOURCES.md
        config/README.md
        config/aes_sources.ini
        config/aes_status.ini
        config/document_types.ini
        config/editors.ini
        config/language_types.ini
        config/languoid_levels.ini
        config/macroareas.ini
        config/macroareas/voronoi/africa.geojson
        config/macroareas/voronoi/australia.geojson
        config/macroareas/voronoi/eurasia.geojson
        config/macroareas/voronoi/north_america.geojson
        config/macroareas/voronoi/papunesia.geojson
        config/macroareas/voronoi/south_america.geojson
        config/med_types.ini
        config/publication.ini
        faq.md
        languoids/README.md
        languoids/dialects.md
        languoids/dialects_3.md
        languoids/dialects_a.md
        languoids/dialects_b.md
        languoids/dialects_c.md
        languoids/dialects_d.md
        languoids/dialects_e.md
        languoids/dialects_f.md
        languoids/dialects_g.md
        languoids/dialects_h.md
        languoids/dialects_i.md
        languoids/dialects_j.md
        languoids/dialects_k.md
        languoids/dialects_l.md
        languoids/dialects_m.md
        languoids/dialects_n.md
        languoids/dialects_o.md
        languoids/dialects_p.md
        languoids/dialects_q.md
        languoids/dialects_r.md
        languoids/dialects_s.md
        languoids/dialects_t.md
        languoids/dialects_u.md
        languoids/dialects_v.md
        languoids/dialects_w.md
        languoids/dialects_x.md
        languoids/dialects_y.md
        languoids/dialects_z.md
        languoids/families.md
        languoids/families_a.md
        languoids/families_b.md
        languoids/families_c.md
        languoids/families_d.md
        languoids/families_e.md
        languoids/families_f.md
        languoids/families_g.md
        languoids/families_h.md
        languoids/families_i.md
        languoids/families_j.md
        languoids/families_k.md
        languoids/families_l.md
        languoids/families_m.md
        languoids/families_n.md
        languoids/families_o.md
        languoids/families_p.md
        languoids/families_q.md
        languoids/families_r.md
        languoids/families_s.md
        languoids/families_t.md
        languoids/families_u.md
        languoids/families_v.md
        languoids/families_w.md
        languoids/families_x.md
        languoids/families_y.md
        languoids/families_z.md
        languoids/glottocodes.json
        languoids/languages.md
        languoids/languages_a.md
        languoids/languages_b.md
        languoids/languages_c.md
        languoids/languages_d.md
        languoids/languages_e.md
        languoids/languages_f.md
        languoids/languages_g.md
        languoids/languages_h.md
        languoids/languages_i.md
        languoids/languages_j.md
        languoids/languages_k.md
        languoids/languages_l.md
        languoids/languages_m.md
        languoids/languages_n.md
        languoids/languages_o.md
        languoids/languages_p.md
        languoids/languages_q.md
        languoids/languages_r.md
        languoids/languages_s.md
        languoids/languages_t.md
        languoids/languages_u.md
        languoids/languages_v.md
        languoids/languages_w.md
        languoids/languages_x.md
        languoids/languages_y.md
        languoids/languages_z.md
        languoids/superseded.csv
        languoids/tree/abin1243/md.ini
        languoids/tree/abis1238/md.ini
        languoids/tree/abkh1242/abkh1243/abaz1241/ashk1247/md.ini
        languoids/tree/abkh1242/abkh1243/abaz1241/bezs1238/md.ini
        languoids/tree/abkh1242/abkh1243/abaz1241/md.ini
        languoids/tree/abkh1242/abkh1243/abaz1241/tapa1256/md.ini
        languoids/tree/abkh1242/abkh1243/abkh1244/abzh1238/md.ini
        languoids/tree/abkh1242/abkh1243/abkh1244/bzyb1238/md.ini
        languoids/tree/abkh1242/abkh1243/abkh1244/md.ini
        languoids/tree/abkh1242/abkh1243/abkh1244/samu1242/md.ini
        languoids/tree/abkh1242/abkh1243/md.ini
        languoids/tree/abkh1242/circ1239/adyg1241/abad1240/md.ini
        languoids/tree/abkh1242/circ1239/adyg1241/bezh1247/md.ini
        languoids/tree/abkh1242/circ1239/adyg1241/md.ini
        languoids/tree/abkh1242/circ1239/adyg1241/natu1243/md.ini
        languoids/tree/abkh1242/circ1239/adyg1241/shap1240/md.ini
        languoids/tree/abkh1242/circ1239/adyg1241/temi1249/md.ini
        languoids/tree/abkh1242/circ1239/adyg1241/xaku1238/md.ini
        languoids/tree/abkh1242/circ1239/kaba1278/grea1271/md.ini
        languoids/tree/abkh1242/circ1239/kaba1278/md.ini
        languoids/tree/abkh1242/circ1239/md.ini
        languoids/tree/abkh1242/md.ini
        languoids/tree/abkh1242/ubyk1235/md.ini
        languoids/tree
error: The following untracked working tree files would be removed by checkout:
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/arsi1238/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/bora1272/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/bora1272/yaab1234/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/gabr1255/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/guji1239/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/kere1294/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/saku1238/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/bora1271/sala1277/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/orma1241/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/orma1241/muny1242/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/orma1241/nucl1702/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/orma1241/waat1239/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/cent2424/tana1300/waat1238/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/bare1280/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/nort3272/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/nort3272/raya1242/md.ini
        languoids/tree/afro1255/cush1243/east2699/lowl1267/sout3055/main1283/nucl1701/nucl1736/cent2302/east2652/nort3272/well1239/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/hass1238/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1239/algi1247/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1239/cons1245/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1239/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1239/oran1236/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/alge1240/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/anda1287/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/jude1265/kjal1234/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/jude1265/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/jude1265/wqal1234/md.ini
        languoids/tree/afro1255/semi1276/west2786/cent2236/arab1394/arab1395/nort3191/west3017/magh1239/malt1259/malt1254/gozo1238/md.ini
        la
Aborting'
```

FredericBlum commented:
It seems like you are running into this issue: https://github.com/cldf/cldfbench/issues/89

> I first removed the windows path limitation. I then cloned glottolog and all the other files by setting the configuration option core.longpaths to be true. I then moved directly into the file containing the code I wanted to run, and then I run the command.

https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry

jaimerafaelms committed:
correction on transcriptions

corrections in Shuar, Southern Nambikuara, Urarina

jaimerafaelms committed:
Update TRANSCRIPTION.md

updated TRANSCRIPTION.md

jaimerafaelms commented:
> It seems like you are running into this issue: https://github.com/cldf/cldfbench/issues/89

¡Ahí radicaba el error! Muchas gracias Frederic, he probado ahora y finalmente me funcionó el comando `cldfbench`. He subido commits con unas correcciones y el nuevo `TRANSCRIPTION.md`. 

> Necesito la descripción de esos fonemas para proponer un simbolo adecuado de CLTS. Estoy seguro que lo vamos a encontrar. Pero por solo ver los simbolos sin nada más es dificil.

Veo que en CLTS hay símbolos parecidos a los fonemas que nos faltan:

[tʙ̥]
"a cluster of a voiceless alveolar stop plus a voiceless bilabial stop, released with a trill on both lips" (Gabas 1999:11, although the author uses its own symbol: t͜p̃ ). The nearest symbol I have found in CLTS is: tp (voiceless alveolar-and-bilabial stop consonant).

Los siguientes fonemas son de McWhorter and Good (2012):

[mᵇ]
pre-nasalized voiced bilabial stop 
(also described in the source as: labial nasal consonant with secondary oral stop release)

[nᵈ]
pre-nasalized voiced alveolar stop 
(also described in the source as: alveolar nasal consonant with secondary oral stop release)

[nᶡ]
pre-nasalized voiced palatal stop 
(also described in the source as: palatal nasal consonant with secondary oral stop release)

[ŋᵍ]
pre-nasalized voiced velar stop 
(also described in the source as: velar consonant with secondary oral stop release)

jaimerafaelms committed:
corrected symbols

updated the following phonemes to CLTS symbols:
Karo:
[tʙ̥]
"a cluster of a voiceless alveolar stop plus a voiceless bilabial stop, released with a trill on both lips" (Gabas 1999:11, although the author uses its own symbol: t͜p̃ ). The nearest symbol I have found in CLTS is: tp (voiceless alveolar-and-bilabial stop consonant).

Saramaccan
(McWhorter and Good 2012):

[mᵇ]
pre-nasalized voiced bilabial stop
(also described in the source as: labial nasal consonant with secondary oral stop release)

[nᵈ]
pre-nasalized voiced alveolar stop
(also described in the source as: alveolar nasal consonant with secondary oral stop release)

[nᶡ]
pre-nasalized voiced palatal stop
(also described in the source as: palatal nasal consonant with secondary oral stop release)

[ŋᵍ]
pre-nasalized voiced velar stop
(also described in the source as: velar consonant with secondary oral stop release)

FredericBlum committed:
remove duplicates

FredericBlum commented:
Closes #2
